scripts/simulations/simulation_for_1d.py

```diff
@@ -102,21 +102,54 @@
                                  CTCF_right_positions, 
                                  **paramdict)
 
-with h5py.File(folder+"/LEFPositions.h5", mode='w') as myfile:
+with h5py.File(folder+"/LEFPositions.h5", mode = 'w') as myfile:
+    # creating data set for LEF positions
     dset = myfile.create_dataset("positions", 
```

Comment:
now that we have more  keys in the h5py file we may want to think of more clear names for them (e.g. dset is not a very distinguishing prefix, and it might be better to call it , e.g. loop_positions)

Reply:
names changed to {dset:dset_loop_positions, dset_possible_right: dset_ctcf_possible_right, dset_pos_right:dset_ctcf_positions_right, etc.,}

Reply:
let's get rid of `ctcfrightlist` as a variable in favor of `CTCF_right_positions` throughout